compiler/damlc/tests/src/DA/Test/DamlcIntegration.hs

```diff
@@ -169,9 +169,10 @@ testCase args version getService outdir registerTODO file = singleTest file . Te
       Compile.unsafeClearDiagnostics service
       ex <- try $ mainProj args service outdir log (toNormalizedFilePath file) :: IO (Either SomeException Package)
       diags <- Compile.getDiagnostics service
+      let diags' = [d | d <- diags, _severity (snd d) /= Just D.DsInfo] -- Discard hlint hints!
```

Comment:
As I mentioned above, if we make turning off hlint an option instead of tryting to filter out HLint diagnostics based on some rather crude heuristics.

Reply:
Yes. Agree.

Reply:
Fixing.